bin/node/runtime/src/lib.rs

```diff
@@ -658,6 +659,10 @@ impl onchain::Config for OnChainSeqPhragmen {
 	>;
 	type DataProvider = <Runtime as pallet_election_provider_multi_phase::Config>::DataProvider;
 	type WeightInfo = frame_election_provider_support::weights::SubstrateWeight<Runtime>;
+
+	// TODO no idea what to use here
+	type MaxBackersPerWinner = ConstU32<{ u32::MAX }>;
```

Comment:
Should be 1024 or something a few orders larger than `MaxNominatorRewardedPerValidator`.